src/main/java/com/dougnoel/sentinel/steps/TableVerificationSteps.java

```diff
@@ -350,4 +352,33 @@ public static void verifyStoredTextRowContainsXpath(String tableName, String key
         }
     }
 
+    /**
+     * Refreshes the page, waiting the configured time for the table to contain the given value in the given row and column.
+     * <p>
+     * <b>Gherkin Examples:</b>
+     * <ul>
+     * <li>I wait up to 15 seconds for the cell in the 1st row and the Status column of the User table to have the text Disabled</li>
+     * <li>I wait up to 5 seconds for the cell in the last row and the Status column of the ID table to not contain the text 123</li>
+     * </ul>
+     * @param numberOfSecondsToWait String number of seconds to wait. Must be an integer. Recommended minimum value: 2 seconds.
+     * @param rowNum String row of the cell to check.
+     * @param columnName String column of the cell to check.
+     * @param tableName String name of the table.
+     * @param assertion String if null is passed, looks for match(es), if any strong value is passed, looks for the value to not exist.
+     * @param matchType String whether we are doing an exact match or a partial match.
+     * @param textToMatch String the text to look for in the column.
+     */
+    @Then("^I wait up to (\\d{1,3}) seconds for the cell in the (la|\\d+)(?:st|nd|rd|th) row and the (.*) column of the (.*?) to( not)? (has|have|contains?) the text (.*?)$")
```

Comment:
Is there a reason you want to wait a different amount of time that the default amount set by the timeout property?

Reply:
For things we expect to take over a minute, I think it makes sense to explicitly state in the step. But if you disagree I could make this use the default timeout and just up that to whatever is necessary for the test.